### ISSUE TITLE
How to plot the loss/P-R curve

### ISSUE THREAD

**Issue by MrunaliniN** [Scalsol/mega.pytorch#104]
Hello there,
How to plot the P-R or loss curve during training or from the log.txt file? 
Without that it appears hard to understand how the training is going.
appreciate your help
